roles/contiv_network/tasks/ovs.yml

```diff
@@ -53,9 +53,16 @@
     - "tcp:127.0.0.1:6640"
     - "ptcp:6640"
 
+- name: check selinux status
+  shell: sestatus | egrep "SELinux status:.*enabled"
+  register: selinux_set
+  when: (ansible_os_family == "RedHat")
```

Comment:
>  It should not break on Ubuntu 

It won't due to `ignore_errors=true` and the conditional to check on the stdout, right?

 > It should execute semanage command only when selinux is not disabled and when OS is redhat.

all this will be be ensured just by the check `selinux_set.stdout | match("SELinux status:.*enabled")`, right?

Actually thinking more, I think `ignore_errors=true` might not be the best way to do this as it will hide errors due to other command execution failures that we haven't envisioned. I think a better approach is following, wdyt?

```
- name: check selinux status
  shell: sestatus
  register: selinux_set
  when: (ansible_os_family == "RedHat")

- name: permit openvswitch_t type in selinux
  shell: >
       semanage permissive -a openvswitch_t
  when: (ansible_os_family == "RedHat") and (selinux_set.stdout | match("SELinux status:.*enabled"))
  become: true
```